src/shared/colors.ts

```diff
@@ -15,6 +15,7 @@ enum Colors {
   BODY_TEXT = '#575757',
   BLACK = '#000000',
   WHITE = '#FFFFFF',
+  PLACEHOLDER = '#b1b1b1',
```

Comment:
**let's not call** COLOR variable just by the place where they used - further it's not comfortable to re-use them, - as they lose their initial meaning. (placeholder could be any color - LIGHT_GREY brings right sence).

This color is already used in master like LIGHT_GREY - please reuse it.

Reply:
Done!